cicd/pipelines/working-with-pipelines-web-console.adoc

```diff
@@ -0,0 +1,49 @@
+:_content-type: ASSEMBLY
+[id="working-with-pipelines-web-console"]
+= Working with {pipelines-title} in the web console
+include::_attributes/common-attributes.adoc[]
+:context: working-with-pipelines-web-console
+
+toc::[]
+
+You can use the *Administrator* or *Developer* perspectives to create and modify `Pipeline`, `PipelineRun`, and `Repository` objects from the *Pipelines* page in the {product-title} web console.
+You can also use the *+Add* page in the *Developer* perspective of the web console to create CI/CD pipelines for your software delivery process.
+
+// Dev console
+include::modules/op-odc-pipelines-abstract.adoc[leveloffset=+1]
+
+[discrete]
+[id="prerequisites_working-with-pipelines-web-console"]
+=== Prerequisites
```

Comment:
A level 3 heading right after a level 1 heading?

Reply:
(Even if it is justified, I have a suspicion this might break portal builds, so if you need to keep it we probably want a portal test)

Reply:
I think it's just a typo, no big deal